docs/reference/deploy/master-config-reference.rst

```diff
@@ -1163,6 +1177,54 @@ those partitions/queues.
    the HPC partition named ``defq_GPU`` with the ``gpu_type`` property set, and Slurm constraint
    associated with the feature ``XL675d`` used to identify the model type of the compute node.
 
+**********************************
+ ``additional_resource_managers``
+**********************************
+
+For Kubernetes, cluster administrators can define additional resource managers to connect the
+Determined master service to remote clusters. For a single resource manager or to define the default
+resource manager, do not define this under ``additional_resource_manager``; instead, use the primary
+``resource_manager`` key.
```

Comment:
For Kubernetes installations, cluster administrators can define additional resource managers to connect the Determined master service to remote clusters. To define a single resource manager or to set the default resource manager, do not define this under ``additional_resource_manager``; instead, use the primary ``resource_manager`` key.